tasks/cephfs/test_auto_repair.py

```diff
@@ -24,48 +24,35 @@ def test_backtrace_repair(self):
         MDS should verify/fix backtrace on fetch dirfrag
         """
 
-        # trim log segment as fast as possible
-        self.set_conf('mds', 'mds cache size', 100)
-        self.set_conf('mds', 'mds verify backtrace', 1)
-        self.fs.mds_fail_restart()
-        self.fs.wait_for_daemons()
-
-        create_script = "mkdir {0}; for i in `seq 0 500`; do touch {0}/file$i; done"
-        # create main test directory
-        self.mount_a.run_shell(["sudo", "bash", "-c", create_script.format("testdir1")])
-
-        # create more files in another directory. make sure MDS trim dentries in testdir1
-        self.mount_a.run_shell(["sudo", "bash", "-c", create_script.format("testdir2")])
+        self.mount_a.run_shell(["sudo", "mkdir", "testdir1"])
+        self.mount_a.run_shell(["sudo", "touch", "testdir1/testfile"])
+        dir_objname = "{:x}.00000000".format(self.mount_a.path_to_ino("testdir1"))
 
         # drop inodes caps
         self.mount_a.umount_wait()
 
         # flush journal entries to dirfrag objects
         self.fs.mds_asok(['flush', 'journal'])
+        self.fs.mds_stop()
 
-        self.mount_a.mount()
-        self.mount_a.wait_until_mounted()
-
-        # wait MDS to trim dentries in testdir1. 60 seconds should be long enough.
-        time.sleep(60)
+        # drop the journal so that cache will be empty on MDS restart
+        self.fs.journal_tool(["journal", "reset", "--force"])
```

Comment:
You're right, I forgot that the header is guaranteed to be written during flush.  Updated patch.